sdk/search/Azure.Search.Documents/api/Azure.Search.Documents.netstandard2.0.cs

```diff
@@ -271,19 +270,18 @@ public partial class Autocompletion
     }
     public partial class AzureActiveDirectoryApplicationCredentials
```

Comment:
@heaths - any chance we can hide this and use ClientSecretCredential instead?  We should at least call it `Credential` and throw a `Search` in there for good measure if not.

Reply:
#10815 

I haven't had a chance to tweak all the generated models yet. Was focusing on what we needed for last Friday's meeting. Once I can pivot back to Search I can make these updates.